build/actions.sh

```diff
@@ -12,6 +12,7 @@ set -o errexit
 source build/common.sh
 
 write-release-date() {
+  mkdir -p _build
```

Comment:
The Makefile is supposed to do this on line 71:

$(shell mkdir -p _bin _release _tmp _build/hello _build/oil)

But I think I am going to use the more consistent style having shell functions write to stdout, and then the Makefile should redirect to `> $@` (i.e. the output file).  Right now _build/release-date.txt is unnecessarily repeated in the shell script and Makefile.